addons/mail/static/src/core/message_model.js

```diff
@@ -66,6 +67,8 @@ export class Message {
     notificationType;
     /** @type {string} */
     pinned_at;
+    /** @type {string} */
+    writeDate;
```

Comment:
Can just name it `write_date`, so it's the exact same naming as server-side, and you don't need to put some LOCs for renaming.